repos/system_upgrade/el7toel8/actors/scanmemory/tests/test_scanmemory.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from leapp.libraries.actor import library
+from leapp.libraries.common import testutils
+from leapp.libraries.stdlib import api
+from leapp.models import MemoryInfo
+
+
+class mocked_get_memoryinfo(object):
+    def __init__(self, filename):
+        self.filename = os.path.join('tests', 'files', filename)
```

Comment:
As @pirat89 already mentioned you don't need to create a test file and then parse it to test proper info is produced.

If you consider going with current approach (testing based on files) you should not mock library._get_memoryinfo but rather mock the open function to feed it test file as /proc/meminfo

Reply:
@fernflower It's small misunderstanding here. My note was related to tests of the `checkmemory` actor, which is not working with files at all and it is just consuming information provided by the scanmemory actor. In this case, I am ok with test files.

Reply:
@pirat89 Still if test files are to be kept in place the mocking of the whole _get_memory_info function is wrong